lib/components/viewers/RouteRow.js

```diff
@@ -1,10 +1,10 @@
-import React, { PureComponent } from 'react'
-import styled from 'styled-components'
 import { Label, Button } from 'react-bootstrap'
 import { VelocityTransitionGroup } from 'velocity-react'
+import React, { PureComponent } from 'react'
+import styled from 'styled-components'
```

Comment:
Sort imports

Reply:
Sorted in 
092b53f aside from React which I think needs to come first